examples/auth_example/auth_example_client/lib/src/protocol/protocol.dart

```diff
@@ -41,7 +41,10 @@ class Protocol extends _i1.SerializationManager {
   }
 
   @override
-  String? getClassNameForObject(Object data) {
+  String? getClassNameForObject(Object? data) {
+    if (data == null) return 'null';
```

Comment:
Chore: Remove this check from the generate code.

We already validate this case inside of `getClassNameForObject`. We don't need it here.